io.github.mrmyhuang.cbetar2.desktop

```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
```

Comment:
this file should be submitted upstream to be handle there. A Linux desktop app that doesn't have a .desktop, it's a bug.

Reply:
fix